main.js

```diff
@@ -22,16 +26,17 @@ function updateRange() {
 
   // Validate & set values
   if (minRange.value !== '' && maxRange.value !== '') {
+    // Set ranges to current ranges section
     currentMin.innerText = minRange.value;
     currentMax.innerText = maxRange.value;
-    updateRangeBtn.classList.add('enable');
   }
 
+  // Add error class for styling inputs
   if (minRange.value === '') {
-    minRange.style.border = "2px solid red";
+    minRange.classList.add('error');
   }
 
   if (maxRange.value === '') {
-    maxRange.style.border = "2px solid red";
+    maxRange.classList.add('error');
```

Comment:
I think it was a good call tp change this to a class as opposed to using the style method. My mentor told me to avoid using style in JS as much as possible (though I'm not positive why).

Reply:
I agree, I think this just helps us add/remove a set of styles much easier than using multiple js .style.display = "blah blah" declarations.